interface/billing/billing_report.php

```diff
@@ -1399,6 +1421,14 @@ function MarkAsCleared(Type) {
     $('#update-tooltip').tooltip({title: "<?php echo xla('Click Update List to display billing information filtered by the selected Current Criteria'); ?>"});
 });
 </script>
+<script>
+    <?php
+    // jQuery script to change state dynamically
+    $user_settings_php_path = '../../library/ajax/user_settings.php';
```

Comment:
We need to figure out how to remove the above (I'll try to play around with this also )
:)